markup-workflow.md

```diff
@@ -0,0 +1,45 @@
+# Markup workflow
+
+Markup is the process which we take design sketch and convert them to Html/Css files.  
+The purpose of this chapter is to understand the process of creating markup with outsourcing providers.
+
+Prepare the instruction file
+
+Create an explanation file contains the design sketch and instructions to the Markup developer.  
+you can see example [here](https://docs.google.com/spreadsheets/d/1GUch-TRll47WAbuLORqHeElqrEHolMTPg6kFnU97ItY/edit#gid=0).
+
+* Be sure to consider all elements of the sketch.
+* If there is a separate design for mobile, be sure to refer to it.
+* If there are already known css classes, mention them, so the markup developer can use them.
+* Point out the various devices which need to be checked.
+
+### Post the job offer
+
+Post the job offer at [Upwork](https://www.upwork.com/c/3474169/jobs/new).
+
+The job offer should contain:
+
+* Explanation file.
+* PDF file\(s\) contain design screenshots.
+* PSD file contains design layers.
+* [Markup guidelines](https://docs.google.com/document/d/18mDgml5u-LII6YYS0c0D3koVjVRZe3FUCYLcaaJSKhA/edit)
+
+### The Markup developer start working
+
+The developer who takes the jobs start working on it and ask for clarification if needed.  
+When they finished, they post the Html/Css files.
+
+### QA - check the work
+
+Deliver the Html/Css files to our QA staff, and they continue the communication with the Markup developer until the job is done to our satisfaction.
+
+### Approve the work
+
+When the work is done, approve it in Upwork so the developer will get payment.
+
+Before approving the work, check the list below:
+
+* We got all the Html/Css files \(not just link to them\).
+* 
```

Comment:
@liatsade relevant classes, ids, brand/project guideline (or styleguide) relevancy, select field's dynamic/static content (i.e. Country names - standard or out of a unique list), devices to check, define a time box for the entire task, define a timebox for the first feedback and expectation of the following communication (by time and method).